src/main.ts

```diff
@@ -83,13 +89,20 @@ export async function run(): Promise<void> {
       const octokit = github.getOctokit(githubToken);
       const output = JSON.parse(fs.readFileSync(outputFile, 'utf8'));
       const modifiedFiles = promptFiles.join(', ');
-      const body = `⚠️ LLM prompt was modified in these files: ${modifiedFiles}
+      let body = `⚠️ LLM prompt was modified in these files: ${modifiedFiles}
 
 | Success | Failure |
 |---------|---------|
 | ${output.results.stats.successes}      | ${output.results.stats.failures}       |
 
-**» [View eval results](${output.shareableUrl}) «**`;
+`;
+      if (output.sharableUrl) {
```

Comment:
Noticed the `sharableUrl` typo here. I've properly typed the output on `main`, so if we rebase and fix this error, we should be good to merge.

Reply:
I fixed it!